docs/dictionary.rst

```diff
@@ -0,0 +1,1101 @@
+	* :ref:`dict Docker image`
+	* :ref:`dict Dockerfile`
+	* :ref:`dict Dockstore CLI`
+	* :ref:`dict Dockstore GitHub App`
+	* :ref:`dict DOI`
+	* :ref:`dict DS-I Africa`
+	* :ref:`dict EC2`
+	* :ref:`dict egress`
+	* :ref:`dict eLwazi`
+	* :ref:`dict entry`
+	* :ref:`dict faceted search`
+	* :ref:`dict FAIR`
+	* :ref:`dict GA4GH`
+	* :ref:`dict Galaxy`
+	* :ref:`dict Galaxy workflow`
+	* :ref:`dict GCP`
+	* :ref:`dict Gen3`
+	* :ref:`dict GitHub App registration`
+	* :ref:`dict GitHub App tool`
+	* :ref:`dict GitHub App workflow`
+	* :ref:`dict immutable`
+	* :ref:`dict interoperable`
+	* :ref:`dict JSON`
+	* :ref:`dict Jupyter`
+	* :ref:`dict kernel`
+	* :ref:`dict Kids First`
+	* :ref:`dict labels`
+	* :ref:`dict launch with`
+	* :ref:`dict layer`
+	* :ref:`dict legacy registration`
+	* :ref:`dict legacy tool`
+	* :ref:`dict legacy workflow`
+	* :ref:`dict NCI`
+	* :ref:`dict NCPI`
+	* :ref:`dict Nextflow`
+	* :ref:`dict NFL`
+	* :ref:`dict NHGRI`
+	* :ref:`dict NHLBI`
+	* :ref:`dict NIH`
+	* :ref:`dict OICR`
+	* :ref:`dict ORCID`
+	* :ref:`dict organization`
+	* :ref:`dict parameter file`
+	* :ref:`dict preemptible`
+	* :ref:`dict Seven Bridges`
+	* :ref:`dict Spot Instance`
+	* :ref:`dict Terra`
+	* :ref:`dict TES`
+	* :ref:`dict tool`
+	* :ref:`dict UCSC`
+	* :ref:`dict VM`
+	* :ref:`dict WDL`
+	* :ref:`dict workflow`
+	* :ref:`dict Workflow Description Language`
+	* :ref:`dict Workflow Execution Service`
+	* :ref:`dict YAML`
+
+.. _dict .dockstore.yml:
+
+.dockstore.yml
+--------------
+	This file is part of :ref:`dict GitHub App registration`. It indexes workflows or tools within a repository, including their optional test parameter files, and the author(s) of said workflows or tools.  
+
+Further reading: `<https://docs.dockstore.org/en/stable/assets/templates/template.html>`_  
+
+.. updated 2022-05-16  
+
+
+
+.. _dict AnVIL Project:
+
+AnVIL Project
+-------------
+*abbreviation for* Analysis Visualization and Informatics Labspace  
+
+	A cloud-based ecosystem funded by :ref:`dict NHGRI`, bringing together Dockstore, :ref:`dict Gen3`, :ref:`dict Terra`, :ref:`dict NCPI`, :ref:`dict Galaxy`, :ref:`dict Jupyter`, Seqr, and Bioconductor into an integrated platform. Sometimes referred to as just "the AnVIL" or "AnVIL".  
+
+Further reading: `<https://anvilproject.org/>`_  
+
+.. updated 2022-05-16  
+
+
+
+.. _dict API:
+
+API
+---
+*abbreviation for* Application Programmer Interface  
+
+	A software-based intermediary used to exchange data, often between two different platforms. Communication between different cloud platforms is mediated by various APIs, such as :ref:`dict TES`.  
+
+
+.. updated 2022-05-16  
+
+
+
+.. _dict AWS:
+
+AWS
+---
+*abbreviation for* Amazon Web Services  
+
+	A provider of cloud services, most notably cloud computing and cloud storage, available on-demand and hosted by Amazon. :ref:`dict Seven Bridges` is an example of a system that is powered by AWS, and can launch workflows on :ref:`dict EC2` instances.  
```

Comment:
I was using Seven Bridges in order to contrast GCP/Terra, but you're right, this might lead to some confusion even though the SB entry does note it can use GCP too. 

In response to other feedback, the AWS entry currently sits at:

> A provider of cloud services, most notably cloud computing and cloud storage, available on-demand and hosted by Amazon. Netflix and AirBnB are examples of a system that is powered by AWS. Some bioinformatics systems such as [Seven Bridges] can leverage AWS by launching workflows on [EC2] instances.

...which no longer implies SB relies upon AWS, but rather *can* use it. Do you think that is sufficient or should it be excised completely? I think a bioinformatics example is needed, but I also think WES may be a little too obscure to be our example... but maybe there is another option besides AWS, or maybe WES is the way to go here.

Reply:
This new phrasing sounds good to me!